### PR TITLE
Add IMDbOT API

### PR DESCRIPTION
**Pull request by SpEcHiDe**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

### CODE REVIEW

**Comment by kevinunitec** [comment]
I don't want to receive email

File: README.md [hunk 1]
```diff
@@ -1234,6 +1234,7 @@ API | Description | Auth | HTTPS | CORS |
 | [ErosNow](https://pwaproxy.erosnow.com/api/v2/catalog/playlists-all?img_quality=1&location=3&page=121&resp=basic&new=true&start=2&content_limit=10&rows=5&optimized=true&progress=all&watchlist=all&country=US) | API for Bollywood films and many more | No | Yes | Unknown |
 | [Final Space](https://finalspaceapi.com/docs/) | Final Space API | No | Yes | Yes |
 | [Game of Thrones Quotes](https://gameofthronesquotes.xyz/) | Some Game of Thrones quotes | No | Yes | Unknown |
+| [IMDb](https://github.com/SpEcHiDe/IMDbOT) | Unofficial IMDb Movie / Series Information | No | Yes | Yes |
```
Comment (matheusfelipeog) [inline README.md:1237]: Change the title to IMDbOT, please:

```suggestion
| [IMDbOT](https://github.com/SpEcHiDe/IMDbOT) | Unofficial IMDb Movie / Series Information | No | Yes | Yes |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @SpEcHiDe ;)

Can you resolve this review, please?

Also, I tried using this API for testing, but I was unsuccessful. Can you explain how to use it?

**Comment by SpEcHiDe** [comment]
Hi, 

you can check the URLs available in the Wiki Pages, 

Example: https://imdbot.vercel.app/?q=avengers%20end%20game


An example repository which is using the api is: https://github.com/SpEcHiDe/Media-Search-bot/blob/master/bot/sessalc/search_imd_b.py#L35

**Comment by matheusfelipeog** [comment]
> Hi,
> 
> you can check the URLs available in the Wiki Pages,
> 
> Example: https://imdbot.vercel.app/?q=avengers%20end%20game
> 
> An example repository which is using the api is: https://github.com/SpEcHiDe/Media-Search-bot/blob/master/bot/sessalc/search_imd_b.py#L35

Very good. Thanks for showing me

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok ✔

A big THANKS for your update 😄

**Pull request merged**